src/components/WinrateGraph.js

```diff
@@ -3,20 +3,32 @@ const {h, Component} = require('preact')
 const helper = require('../modules/helper')
 const setting = remote.require('./setting')
 
+let winrateGraphMinHeight = setting.get('view.winrategraph_minheight')
 let winrateGraphHeight = setting.get('view.winrategraph_height')
 
 class WinrateGraph extends Component {
     constructor() {
         super()
 
+        this.state = {
+            sidebarWinrateGraphSplit: setting.get('view.winrategraph_height'),
+            sidebarWinrateGraphSplitTransition: true
+        }
+
         this.handleMouseDown = evt => {
             this.mouseDown = true
             document.dispatchEvent(new MouseEvent('mousemove', evt))
         }
+
+        this.handleHorizontalResizerWinrateMouseDown = ({button}) => {
+            if (button !== 0) return
+            this.horizontalResizerWinrateMouseDown = true
+        }
     }
 
     shouldComponentUpdate({width, currentIndex, data}) {
         return width !== this.props.width
+            || winrateGraphHeight !== this.state.sidebarWinrateGraphSplit
```

Comment:
This is somewhat misleading. What you mean is probably:

~~~js
shouldComponentUpdate({width, currentIndex, data}, {sidebarWinrateGraphSplit}) {
    ...
        || sidebarWinrateGraphSplit !== this.state.sidebarWinrateGraphSplit
    ...
~~~

This will check whether the `sidebarWinrateGraphSplit` has changed or not.